test/functionalTest/cases/2193_v1_reply_to_v2_response_on_error_with_cpr/forward_using_GET_v2_entities.test

```diff
@@ -0,0 +1,220 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Due to the way `make coverage` works, all .test files needs a different name (although they are located in different directories). Thus, just removing the sufix is not enough (as the name will be equal to the one used by the NGSIv1 variant and it would colide), prefixing would be needed.